system-contracts/contracts/EvmInterpreterFunctions.template.yul

```diff
@@ -1235,12 +1189,7 @@ function isAddrEmpty(addr) -> isEmpty {
 }
 
 function _fetchConstructorReturnGas() -> gasLeft {
-    //selector is 0x24e5ab4a
-
-    mstore8(0, 0x24)
-    mstore8(1, 0xe5)
-    mstore8(2, 0xab)
-    mstore8(3, 0x4a)
+    mstore(0, 0x24E5AB4A00000000000000000000000000000000000000000000000000000000)
```

Comment:
@jrchatruc Is there a possibility that this will override some bytes that are important before the call?

Reply:
There is no problem, all test pass when switching to `mstore`